battleGame/Intermediate/Config/CoalescedSourceConfigs/Engine.ini

```diff
@@ -1072,8 +1072,8 @@ MaxObjectsNotConsideredByGC=0
 SizeOfPermanentObjectPool=0
 AsyncIOBandwidthLimit=0
 Paths=../../../Engine/Content
-Paths=D:/UnrealProj/battlegame/battleGame/Content
-CutdownPaths=D:/UnrealProj/battlegame/battleGame/CutdownPackages
+Paths=../../../../../../Users/Kevin/Documents/battleGame/battleGame/Content
+CutdownPaths=../../../../../../Users/Kevin/Documents/battleGame/battleGame/CutdownPackages
 ZeroEngineVersionWarning=True
```

Comment:
So this is going to be a potential problem. I mean, we could just make sure all our directories structures are the same, but that might not be possible (especially if Brandon wants to code on his Mac). This isn't throwing a merge conflict either. We will have to think about this.

Edit: Hows this for a solution: Environment variables! Everybody has an environment variable like $PROJECT_PATH that is different for all of us, but will be the same for these types of files.